static/js/util/profile_edit.js

```diff
@@ -196,19 +196,19 @@ export function boundDateField(keySet: string[], label: string, omitDay: boolean
     let validatedDay = 1;
     if (!omitDay) {
       validatedDay = validateDay(newEdit.day);
-      newEdit.day = validatedDay === undefined ? '' : String(validatedDay);
+      newEdit.day = String(validatedDay);
```

Comment:
Nevermind, it looks like the validate functions are changed to never return undefined